clinvar-variant-types/clinvar-variant-types.py

```diff
@@ -17,9 +29,37 @@ def add_transitions(transitions_counter, transition_chain):
         transitions_counter[(transition_from, transition_to)] += 1
 
 
+def find_attribute(rcv, xpath, attribute_name):
+    """Find an attribute in the RCV record which can have either zero or one occurrence. Return a textual representation
+    of the attribute, including special representations for the case of zero or multiple, constructed using the
+    attribute_name parameter."""
+
+    attributes = rcv.findall(xpath)
+    if len(attributes) == 0:
+        return '{} missing'.format(attribute_name)
+    elif len(attributes) == 1:
+        return attributes[0].text
+    else:
+        return '{} multiple'.format(attribute_name)
+
+
+def clin_sig_color(clinical_significance):
```

Comment:
I'm curious where is this function used, I haven't been able to find usages.

Reply:
It isn't! I forgot to remove it, thank you for noticing